core/shared/src/main/scala/zio/Chunk.scala

```diff
@@ -36,7 +37,7 @@ sealed trait Chunk[+A] extends ChunkLike[A] { self =>
   /**
    * Appends an element to the chunk
    */
-  final def +[A1 >: A](a: A1): Chunk[A1] =
+  def +[A1 >: A](a: A1): Chunk[A1] =
```

Comment:
Should we call this `:+` for consistency with `Vector`?

Reply:
Yes I think so. That will also set us up to do the same thing for prepend.